src/xtgeo/well/_well_oper.py

```diff
@@ -493,3 +523,35 @@ def _growfilter(bseries, nleft):
     bseries = transitions | transitions.shift(-1)
 
     return _growfilter(bseries, nsamples - 1)
+
+
+def _get_bseries_by_distance(depth, inseries, distance):
```

Comment:
Is bseries short for bool series?

Reply:
Yes

Reply:
Would perhaps be better being written out (bseries means something else in my head, but should vary with every head)